packages/packages_by_dep_dag.txt

```diff
@@ -1,3 +1,4 @@
+release_util
```

Comment:
If it doesn't depend on significant portions of Galaxy I have a strong preference that it live outside of Galaxy. This depends on less than Ephemeris and Planemo and Pulsar - etc..

Reply:
Yep, completely agree, that's much less awkward